src/base/rss/rss_session.cpp

```diff
@@ -102,6 +102,24 @@ Session::Session()
         m_refreshTimer.start(m_refreshInterval * MsecsPerMin);
         refresh();
     }
+
+    // remove legacy/corrupted settings
```

Comment:
```c++
// remove legacy/corrupted settings
// at least on Windows, QSettings is case-insensitive
// and it can get confused when asked about settings that differ only in their case.
```